File: apps/myjobhunter/backend/app/schemas/profile/skill.py
"""Skill schemas — Phase 1 stub."""
import uuid
from datetime import datetime

from pydantic import BaseModel


class SkillRead(BaseModel):
    id: uuid.UUID
    profile_id: uuid.UUID
    name: str
    years_experience: int | None
    category: str | None
    created_at: datetime

    model_config = {"from_attributes": True}

File: apps/myjobhunter/frontend/src/types/skill/skill.ts
/**
 * Skill as returned by the backend.
 * Mirrors SkillResponse in backend/app/schemas/profile/skill_response.py.
 */
export interface Skill {
  id: string;
  user_id: string;
  profile_id: string;
  name: string;
  years_experience: number | null;
  category: string | null;
  created_at: string;
  updated_at: string;
}
